### PR TITLE
Tips app: Creating tips data model

### PR DESCRIPTION
**Pull request by orIsrael**
If applies, it will generate the Django tips model for the project

### CODE REVIEW

**Comment by ShellyGolden** [comment]
Why did you open a new PR?
You already had it https://github.com/redhat-beyond/Xpense/pull/49

**Comment by orIsrael** [comment]
> Why did you open a new PR? You already had it #49

My branch was broken and I had to reopen it

File: expenses/migrations/0003_alter_expenses_date.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Generated by Django 4.0.3 on 2022-04-11 19:56
```
Comment (ronyyosef) [inline expenses/migrations/0003_alter_expenses_date.py:1]: I think @Yoavsc129 solves that problem, Please use it and don't create a new alter file

File: expenses/migrations/0002_alter_expenses_date.py [hunk 2]
```diff
@@ -0,0 +1,20 @@
+# Generated by Django 4.0.3 on 2022-04-11 15:25
+
```
Comment (ronyyosef) [inline expenses/migrations/0002_alter_expenses_date.py:2]: please don't create alter files for the expenses app

File: tips/migrations/0002_alter_tip_author_alter_tip_text.py [hunk 3]
```diff
@@ -0,0 +1,23 @@
+# Generated by Django 4.0.3 on 2022-04-11 19:18
+
+from django.db import migrations, models
+
```
Comment (ronyyosef) [inline tips/migrations/0002_alter_tip_author_alter_tip_text.py:4]: same here, please rerun make makemigrations so we will have only the init file

**Review by ronyyosef — CHANGES_REQUESTED** [review]
Left some comments, the code here doesn't really match the code that was in the old PR, please make sure all the comments from the old PR are resolved

**Review by ShellyGolden — COMMENTED** [review]
Change the title of the PR

File: expenses/migrations/0001_initial.py [hunk 4]
```diff
@@ -1,4 +1,4 @@
-# Generated by Django 4.0.3 on 2022-04-13 14:31
+# Generated by Django 4.0.3 on 2022-04-18 08:39
```
Comment (ShellyGolden) [inline expenses/migrations/0001_initial.py:1]: This file is already in the main code. 
why did you create it again?

**Review by ronyyosef — CHANGES_REQUESTED** [review]
Only Shelly comment and I think we are ready to merge after that

File: expenses/migrations/0001_initial.py (hunk 4 above)
Comment (orIsrael) [reply expenses/migrations/0001_initial.py]: I think I fixed it

File: expenses/migrations/0002_alter_expenses_date.py (hunk 2 above)
Comment (orIsrael) [reply expenses/migrations/0002_alter_expenses_date.py]: Got it

File: expenses/migrations/0003_alter_expenses_date.py (hunk 1 above)
Comment (orIsrael) [reply expenses/migrations/0003_alter_expenses_date.py]: Done

**Review by ShellyGolden — APPROVED** [review]

**Review by Yoavsc129 — APPROVED** [review]

**Review by svatares — APPROVED** [review]

**Pull request merged**